docs/running_a_server/whitelisting.md

```diff
@@ -0,0 +1,163 @@
+---
+id: whitelisting
```

Comment:
```suggestion
id: whitelist
```

Could it just be that? It'll make it easier to look up directly without having to search